avocado.spec

```diff
@@ -91,6 +120,46 @@ arbitrary filesystem location.
 %files plugins-output-html
 %{python_sitelib}/avocado_result_html*
 
+%package plugins-runner-remote
+Summary: Avocado Runner for Remote Execution
+Requires: avocado == %{version}, fabric
+
+%description plugins-runner-remote
+Allows Avocado to run jobs on a remote machine, by means of an SSH
+connection.  Avocado should be previously installed on the remote machine.
+
+%files plugins-runner-remote
+%{python_sitelib}/avocado_runner_remote*
+
+%package plugins-runner-vm
+Summary: Avocado Runner for libvirt VM Execution
+Requires: avocado == %{version}, avocado-plugins-runner-remote == %{version}
+Requires: libvirt-python
+
+%description plugins-runner-vm
+Allows Avocado to run jobs on a libvirt based VM, by means of
+interaction with a libvirt daemon and an SSH connection to the VM
+itself.  Avocado should be previously installed on the VM.
+
+%files plugins-runner-vm
+%{python_sitelib}/avocado_runner_vm*
+
+%package plugins-runner-docker
+Summary: Avocado Runner for Execution on Docker Containers
+Requires: avocado == %{version}, avocado-plugins-runner-remote == %{version}
+Requires: aexpect
+# FIXME: add conditional dependency, since python-aexpect is still AFAIK
+# not available on EPEL.
```

Comment:
Well one can always build the RPM from sources and use it while building this one, right?